app/models/persistent_volume_claim.rb

```diff
@@ -0,0 +1,6 @@
+class PersistentVolumeClaim < ActiveRecord::Base
```

Comment:
ok, so leave it as `PersistentVolumeClaim` to keep it closer to `PersistentVolume`